lib/shopify_app/session/jwt.rb

```diff
@@ -35,7 +35,7 @@ def set_payload
       payload, _ = parse_token_data(ShopifyApp.configuration&.secret, ShopifyApp.configuration&.old_secret)
       @payload = validate_payload(payload)
     rescue *WARN_EXCEPTIONS => error
-      Rails.logger.warn("[ShopifyApp::JWT] Failed to validate JWT: [#{error.class}] #{error}")
+      ShopifyApp::Utils::Logger.warn("[ShopifyApp::JWT] Failed to validate JWT: [#{error.class}] #{error}")
```

Comment:
This is an example where I just found a Rails logger output, so I figured I would change it to our new one.